components/MeetingDetail/MeetingDetail.test.js

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from "@testing-library/react";
+import MeetingDetail from "./MeetingDetail";
+import { data } from "../../data";
+
+test("renders given location from test data", () => {
+  render(<MeetingDetail data={data[0]} />);
```

Comment:
With a test variable, you can be more semantic when passing props instead of `data={data[0]}` (you won't know what's happening here next week if you need to change the test).